<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Чіпсы</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_17">
    <item row="2" column="1">
     <widget class="QLabel" name="labelAddedFactors">
      <property name="text">
       <string>Додані фактори</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBoxFactors">
      <property name="title">
       <string>Фактори</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_18">
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEditAd"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelIndex">
         <property name="text">
          <string>Індекс споживчих витрат (%)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLineEdit" name="lineEditPriceCompetitor"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelPriceCompetitor">
         <property name="text">
          <string>Середня ціна товару у конкурентів (грн)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelPrice">
         <property name="text">
          <string>Ціна товару (грн)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEditIndex"/>
       </item>
       <item row="5" column="1">
        <widget class="QPushButton" name="pushButtonEdit">
         <property name="text">
          <string>Редагувати</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="pushButtonDelete">
         <property name="text">
          <string>Видалити</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEditPrice"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelAd">
         <property name="text">
          <string>Витрати на рекламу (тис. грн)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Додати</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTableWidget" name="tableWidgetFactors"/>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonCalculate">
      <property name="text">
       <string>Розрахувати</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_19">
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEditFileDirection"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonSelectFile">
        <property name="text">
         <string>Обрати файл</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelCsvFile">
        <property name="text">
         <string>Вкажіть шлях до .csv файлу</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonImport">
        <property name="text">
         <string>Імпортувати</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
